src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -317,9 +317,9 @@ protected function compileComponentTags($value)
             return $value;
         }
 
-        return (new ComponentTagCompiler(
-            $this->classComponentAliases, $this
-        ))->compile($value);
+        return app(ComponentTagCompiler::class, [
```

Comment:
You can't call the app global function from the view component.

Reply:
Ahh, thought it looks weird. :D Is `Container::getInstance()->make()` better? What would be the preferred way?